application/heartbeater.go

```diff
@@ -30,7 +30,7 @@ func StartHeartbeatService(sender senders.Sender, application Tags, source strin
 		source:      source,
 		components:  components,
 		ticker:      time.NewTicker(5 * time.Minute),
-		stop:        make(chan bool, 1),
+		stop:        make(chan struct{}),
```

Comment:
why?

Reply:
`struct{}` is the smallest data type available in go with 0 bytes.
https://dave.cheney.net/2014/03/25/the-empty-struct

Reply:
If you're asking why unbuffered, Hearbeater.Close() will block until goroutine exits.

Reply:
so, your are saving 1 byte of memory, and IMO making the code harder to read

Reply:
chan struct{} is very common in Go. It's the recommended way to declare a channel carrying no data (i.e. just used for event signalling)

Reply:
Ok, clear.